app/README.md

```diff
@@ -0,0 +1,29 @@
+# secret_ballot_app
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Run your tests
+```
+npm run test
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
+
+### Customize configuration
```

Comment:
this isn't a relevant section here :)

readme should include what port to expect things on, and top-level directory structure organization